sdk/webpubsub/azure-webpubsub-client/azure/webpubsub/client/_client.py

```diff
@@ -0,0 +1,850 @@
+            **kwargs,
+        )
+
+    def leave_group(self, group_name: str, **kwargs: Any) -> None:
+        """Leave the client from group
+        :param group_name: The group name. Required.
+        :type group_name: str.
+        :keyword int ack_id: The optional ackId. If not specified, client will generate one.
+        """
+
+        def leave_group_attempt():
+            group = self._get_or_add_group(group_name)
+            ack_id = kwargs.pop("ack_id", None)
+            self._send_message_with_ack_id(
+                message_provider=lambda id: LeaveGroupMessage(group=group_name, ack_id=id),
+                ack_id=ack_id,
+                **kwargs,
+            )
+            group.is_joined = False
+
+        self._retry(leave_group_attempt)
+
+    def send_event(
+        self,
+        event_name: str,
+        content: Any,
+        data_type: Union[WebPubSubDataType, str],
+        **kwargs: Any,
+    ) -> None:
+        """Send custom event to server. For more info about event handler in web pubsub, please refer
+        to https://learn.microsoft.com/en-us/azure/azure-web-pubsub/howto-develop-eventhandler
+
+        :param event_name: The event name. Required.
+        :type event_name: str.
+        :param content: The data content that you want to send to event handler that registered in web
+         pubsub. Required.
+        :type content: Any.
+        :param data_type: The data type. Required.
+        :type data_type: Union[WebPubSubDataType, str].
+        :keyword int ack_id: The optional ackId. If not specified, client will generate one.
+        :keyword bool ack: If true, the message won't contains ackId and no AckMessage
+         will be returned from the service.
+        """
+
+        def send_event_attempt():
+            ack = kwargs.pop("ack", False)
+            ack_id = kwargs.pop("ack_id", None)
+            if not ack:
+                self._send_message_with_ack_id(
+                    message_provider=lambda id: SendEventMessage(
+                        data_type=data_type, data=content, ack_id=id, event=event_name
+                    ),
+                    ack_id=ack_id,
+                    **kwargs,
+                )
+            else:
+                self._send_message(
+                    message=SendEventMessage(data_type=data_type, data=content, event=event_name), **kwargs
+                )
+
+        self._retry(send_event_attempt)
+
+    def send_to_group(
+        self,
+        group_name: str,
+        content: Any,
+        data_type: Union[WebPubSubDataType, str],
+        **kwargs: Any,
+    ) -> None:
+        """Send message to group.
+        :param group_name: The group name. Required.
+        :type group_name: str.
+        :param content: The data content. Required.
+        :type content: Any.
+        :param data_type: The data type. Required.
+        :type data_type: Any.
+        :keyword bool ack: If true, the message won't contains ackId and no AckMessage
```

Comment:
Why when `ack` is true, it won't contains `ackId`?

Reply:
python rename `fire_and_forget` to `ack` and `fire_and_forget=true` shall equal to `ack=false`. I forget to update the description.
Updated.